lib/spack/spack/store.py

```diff
@@ -33,10 +34,76 @@
 import spack.config
 import spack.util.path
 import spack.database
-import spack.directory_layout as dir_layout
+import spack.directory_layout
+
 
 #: default installation root, relative to the Spack install path
-default_root = os.path.join(spack.paths.opt_path, 'spack')
+default_install_root = os.path.join(spack.paths.opt_path, 'spack')
```

Comment:
`default_install_tree_root` just for clarity?

Reply:
The code gets harder to read when we have to split lines for flake constantly, I'm a little hesitant to make a long variable name longer.